.github/workflows/analyze.yml

```diff
@@ -0,0 +1,16 @@
+name: Analyze
+on: [push]
+jobs:
+    analyze:
+        runs-on: ubuntu-20.04
+        steps:
+            - name: Install PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 8.1
+            - name: Checkout code
+              uses: actions/checkout@v3
+            - name: Install PHP dependencies
+              run: composer install --no-interaction --no-progress --no-scripts
+            - name: Run PHP CodeSniffer
+              run: ./vendor/bin/phpstan analyze --no-progress --no-interaction --no-ansi
```

Comment:
The command requires a path:
```php
./vendor/bin/phpstan analyze --no-progress --no-interaction --no-ansi ./some/path
```

Reply:
The paths are set in the phpstan.neon config file. Passing a path in the command line here would override the paths set in the config, so I think it's better managed there.